src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck start working!");
+
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stop working!");
```

Comment:
Similarly, the message 'Truck stop working!' should clearly indicate the machine type. You could rephrase it to 'Truck has stopped working.' to make it more informative.